src/processor/operator/update/set_executor.cpp

```diff
@@ -0,0 +1,68 @@
+#include "processor/operator/update/set_executor.h"
+
+namespace kuzu {
+namespace processor {
+
+void NodeSetExecutor::init(ResultSet* resultSet, ExecutionContext* context) {
+    nodeIDVector = resultSet->getValueVector(nodeIDPos).get();
+    evaluator->init(*resultSet, context->memoryManager);
+    rhsVector = evaluator->resultVector.get();
+}
+
+void SingleLabelNodeSetExecutor::set() {
+    evaluator->evaluate();
+    for (auto i = 0u; i < nodeIDVector->state->selVector->selectedSize; ++i) {
```

Comment:
This is a bit confusing. Can we not directly use existing write interface of Column? so it can be 
```
column->write(nodeIDVector, rhsVector);
```

Also, what do you mean lhs and rhs here? I don't quite grasp where the convention comes from. I feel nodeIDVector and propertyVector is more clear.